src/test/java/com/dsl/clauses/linking/ClauseBuilderTest.java

```diff
@@ -230,22 +230,22 @@ void skipWithVariable() {
     @Test
     void limitWithInt() {
         String s = match()
-            .path(node("n"))
+            .matchPath(node("n"))
             .returns(select("n").prop("name")).limit(1).asString();
         Assertions.assertEquals("MATCH (n) RETURN n.name LIMIT 1", s);
     }
 
     @Test
     void limitWithLiteral() {
         String s = match()
-            .path(node("n")).returns(select("n").prop("name")).limit(literal(1)).asString();
+            .matchPath(node("n")).returns(select("n").prop("name")).limit(literal(1)).asString();
```

Comment:
what's the login behind of this rename?

Reply:
yes, the problem is that the merge/with/create/match have the same logic, and the name cant be the same or it will not work out